src/app/common/services/comment.service.ts

```diff
@@ -51,7 +51,6 @@ export class CommentService {
 
   private handleError<T>(operation = 'operation', result?: T) {
     return (error: any): Observable<T> => {
-      console.error(error);
```

Comment:
I believe we shouldn't just mute error messages.
We need to use front-end logger here and control it's behaviour, so in debug mode it should show errors but in production mode - do not show to console but maybe log them to server.